src/App/App.js

```diff
@@ -56,7 +56,12 @@ class App extends Component {
  
   render() {
     return (
-      <div className="App">
+      <div className="App" 
+        style={{
```

Comment:
This gives size to the outermost container for styling purposes. Again feel free to move it! Or lmk if you did this elsewhere and I will push up a PR w/o it.